test/tests/pbs_smoketest.py

```diff
@@ -1517,11 +1517,11 @@ def test_validate_pbsnodes_attrs(self):
             'sharing', 'last_state_change_time']
         possible_dflt_attrs = {'license': 'l', 'last_used_time': 'arbitrary'}
         self.server.expect(NODE, {'state': 'free'},
-                           id=self.server.shortname, interval=3)
+                           id=self.mom.shortname, interval=3)
```

Comment:
Is this change intended for this PR?

Reply:
Without this change the test will fail if the mom is running in a separate machine/container. I've added this along with the PR to save time as it is a single line change.

Reply:
Since this is PTL bug you could have done it in different PR for better tracking. Since you have already done I am fine with this code as is.